springwolf-core/src/main/java/io/github/stavshamir/springwolf/configuration/DefaultAsyncApiDocketService.java

```diff
@@ -64,19 +64,25 @@ private AsyncApiDocket parseApplicationConfigProperties(SpringwolfConfigProperti
         return builder.build();
     }
 
-    private static Info buildInfo(@Nullable SpringwolfConfigProperties.ConfigDocket.Info info) {
```

Comment:
This method has an input variable `info` and now a new local info variable as return type. I renamed the variables to make clear, which one is the configdocketinfo and which one is the resulting asyncapiInfo Object.